test/io-tests/test_io.py

```diff
@@ -87,6 +87,8 @@ def defaultenv():
         "PGRST_DB_URI": os.environ["PGRST_DB_URI"],
         "PGRST_DB_SCHEMAS": os.environ["PGRST_DB_SCHEMAS"],
         "PGRST_DB_ANON_ROLE": os.environ["PGRST_DB_ANON_ROLE"],
+        "PGRST_DB_LOAD_GUC_CONFIG": "false",
+        "PGRST_LOG_LEVEL": "warn",
```

Comment:
I put a non-existent `root-spec` function in the role settings config. The `wait_until_ready` function then failed, but the logs only showed a couple of 503 and not the 404s. I changed to warn to show this in case a similar error happens.

Reply:
This one broke another test. I've just reverted it.

Reply:
Yes, that confirms my gut feeling, when I first read it. We'll have to add a few tests for different log level somewhere else anyway.